subsys/nrf_security/Kconfig.tls

```diff
@@ -222,6 +222,16 @@ config MBEDTLS_SSL_DTLS_BADMAC_LIMIT
 
 endif # MBEDTLS_SSL_PROTO_DTLS
 
+# This config can be set only for OpenThread's TCAT and CoAPs, because these are isolated cases,
```

Comment:
This config **can** or **has to** be set to make handshake work properly? If it is something we must have, I would recommend selecting mbedtls config for TCAT and COAP. Otherwise, we can leave it as is.

Reply:
good catch! added additional selects and removed default value